issues.md

```diff
@@ -1,156 +1,150 @@
 
-### Task description
+### Reporting an issue
 
-We should therefore replace all instances of the `\{{ warning }}` macro with the following:
+- Depending on the problem you've discovered, report it by filing a [documentation issue](https://github.com/mdn/content/issues), a [localization issue](https://github.com/mdn/translated-content/issues), or a [platform issue](https://github.com/mdn/yari/issues).
 
-> **Warning:** Main subject line
->
-> Details of the warning.
-> It can have multiple paragraphs.
+- To open an issue, use the appropriate template available in the repository. For example, to report a content bug, use the the [Content issue](<https://github.com/mdn/content/issues/new?assignees=&labels=needs+triage&template=content-bug.yml>) template in the `mdn/content` repository.
 
-### Actionable outcome
+- Provide sufficient information while reporting the issue:
 
-- [ ] There are no more instances of the `\{{ warning }}` macro in the `mdn/content` repository.
-- [ ] Deprecate `\{{ warning }}` macro
-- [ ] Notify localization team leads of the change.
-```
-
-## Make progress, not noise
-
-Think carefully about the way you handle communication in the project — make sure it is useful, and that it doesn't make other contributors jobs harder. Submitting pull requests to fix issues is great, but are they truly useful, and easy to review? Filing issues and joining in other conversations is fine, but are your issues and comments on topic, or are they just adding noise?
-
-As a rule, do:
-
-- Use [GitHub discussion](https://github.com/mdn/mdn-community/discussions) before filing an issue. This helps to keep issues focused and productive.
-- Ask questions using other mechanisms like [chat rooms](https://chat.mozilla.org/#/room/#mdn:mozilla.org) or [forums](https://discourse.mozilla.org/c/mdn/236) if you are not sure whether something is useful or have a simple question.
-- Read our [contribution guidelines](/en-US/docs/MDN/Community) and [writing guidelines](/en-US/docs/MDN/Writing_guidelines) first to try to solve the issue yourself.
-
-Don't:
+  - **Issue title** must convey succinctly the _required action_.
 
-- Complicate issues by trying to discuss multiple topics at once, or making off-topic comments.
-- Open lots of issues asking vague questions.
-- Ask questions without trying to solve the problem yourself first.
+  - **Issue description** must clearly describe the bug and the action required to resolve the issue. It must also list the task or sub-tasks to be completed to resolve the issue. Some other guidelines include:
+    - Use the description field to indicate the status of the task or sub-tasks by using checklists.
+    - Updates to the status of a task should be conveyed by editing the task list in the issue description instead of adding a new comment. One should not have to scroll through comments or replies to find the status of a task or sub-task.
```

Comment:
What about something like:

```suggestion
    - Update the status of a task in the issue description instead of commenting on the issue. Use [task lists](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/about-task-lists) if an issue has multiple parts. This helps others who may otherwise need to scroll through comments on the issue to determine its status.
```

Feel free to improve it